package.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
Review how many of the properties in this object actually have meaning for NPM. I'm guessing things like `publisher` and `engines`, among others, only need to be there for VS Code extensions.

Reply:
I'll leave them in based on other examples.